### PR TITLE
add contribution documentation

### PR DESCRIPTION
**Pull request by Yancey1989**

### CODE REVIEW

File: docs/contribution.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    the push allows you to create a new pull request when requesting
+    the [official repo](https://github.com/alibaba/BladeDISC), please
+    follows [this article](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+    to create a pull request.
+
+    If your pull request is fixing an exiting issue, please fill in the key word
+    [fixes #issue-id](https://help.github.com/articles/closing-issues-using-keywords/)
+    to close the issue when the pull request is merged.
+
+    Please feel free to assign a reviewer and assign an asocial label on
```
Comment (linearhit) [inline docs/contribution.md:105]: asocial, typo？

File: docs/contribution.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    ``` bash
+    git push origin new-feature-branch
+    ```
+
+    the push allows you to create a new pull request when requesting
+    the [official repo](https://github.com/alibaba/BladeDISC), please
+    follows [this article](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+    to create a pull request.
+
+    If your pull request is fixing an exiting issue, please fill in the key word
```
Comment (linearhit) [inline docs/contribution.md]: fixing -> to fix

File: docs/contribution.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    you run `git commit` command:
+
+    ``` text
+    ➜  BladeDISC git:(new_feature) ✗ pre-commit run -a
+    copyright_checker........................................................Passed
+    ```
+
+1. Create a Pull Request
+
+    Once finish the new feature or bugfix, you can push the local
```
Comment (linearhit) [inline docs/contribution.md:89]: Once the new feature or the bugfix is done, you can ...

File: docs/contribution.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    work into your forked repo:
+
+    ``` bash
+    git push origin new-feature-branch
+    ```
+
+    the push allows you to create a new pull request when requesting
+    the [official repo](https://github.com/alibaba/BladeDISC), please
+    follows [this article](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
```
Comment (linearhit) [inline docs/contribution.md:98]: follows -> follow

File: docs/contribution.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+1. Create a Pull Request
+
+    Once finish the new feature or bugfix, you can push the local
+    work into your forked repo:
+
+    ``` bash
+    git push origin new-feature-branch
+    ```
+
+    the push allows you to create a new pull request when requesting
```
Comment (linearhit) [inline docs/contribution.md:96]: the -> The

File: docs/contribution.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+1. Clone
+
+    To make a copy of your repo on your host, please run:
+
+    ``` bash
+    git clone git@github.com:your-github-account/BladeDISC.git
+    ```
+
+    please go to [build from source](./build_from_source.md) to check how
```
Comment (linearhit) [inline docs/contribution.md:46]: please -> Please

**Review by linearhit — CHANGES_REQUESTED** [review]

File: docs/contribution.md (hunk 1 above)
Comment (Yancey1989) [reply docs/contribution.md]: thx, done.

**Review by linearhit — APPROVED** [review]
LGTM

**Pull request merged**